.changeset/quiet-paws-study.md

```diff
@@ -0,0 +1,66 @@
+---
+'@backstage/plugin-techdocs-backend': patch
+---
+
+Output logs from a TechDocs build to a logging transport in addition to existing frontend event stream, for capturing these logs to other sources.
+
+This allows users to capture debugging information around why tech docs fail to build without needing to rely on end users ending information from their web browser.
+
+The most common use case is to log to the same place as the rest of the backend application logs.
+
+To use, include
+
+```
+import { DockerContainerRunner } from '@backstage/backend-common';
+import {
+  createRouter,
+  Generators,
+  Preparers,
+  Publisher,
+} from '@backstage/plugin-techdocs-backend';
+import Docker from 'dockerode';
+import { Router } from 'express';
+import { PluginEnvironment } from '../types';
+
+export default async function createPlugin(
+  env: PluginEnvironment,
+): Promise<Router> {
+  // Preparers are responsible for fetching source files for documentation.
```

Comment:
I think remove these comments, and just comment the line where the `buildTransportLogger` is. That makes it clear that that's the interesting part

Reply:
Comments removed, thanks again 🙂